dialectic/src/unary.rs

```diff
@@ -1,15 +1,7 @@
 //! The unary numbers, represented by zero [`Z`] and successor [`S`].
 
-pub mod constants {
-    //! Predefined value-level constants for small type-level numbers. Each of these corresponds
-    //! to a type synonym in [`crate::unary::types`].
-    #![allow(missing_docs)]
-    dialectic_macro::generate_unary_constants!(256);
-}
-
 pub mod types {
```

Comment:
I think it's preferable to remove this module as well, and alter examples/benchmarks/tests which use it to adapt.